rubocop-klaxit/cops/postgres_no_between.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module PostgreSQL
+      # Using BETWEEN in PostgreSQL is a bad practice. We a cop to detect
+      # between keyword in string.
```

Comment:
```suggestion
      # Using BETWEEN in PostgreSQL is a bad practice. It may give false positives.
      # See https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_BETWEEN_.28especially_with_timestamps.29
```